package-lock.json

```diff
@@ -1054,7 +1054,7 @@
     },
     "chalk": {
       "version": "1.1.3",
-      "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
+      "resolved": "http://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```

Comment:
What's the deal with these changes? Is there a reason not to use `https`?

Reply:
🤷🏻‍♂️ npm did this so I committed it. Can remove the change if you think it's best